src/client/pages/OriginalDataPoint/components/NationalClasses/components/NationalClassesTable.tsx

```diff
@@ -23,24 +23,27 @@ export const NationalClassesTable = (props: Props) => {
   const canEdit = useCanEditData(originalDataPoint)
 
   return (
-    <DataGrid gridTemplateColumns={`${print ? `100px ` : ''}minmax(240px, 40%) 1fr`} withActions={canEdit}>
-      {print && (
-        <DataCell gridRow={`1/${nationalClasses.length + 2}`} header lastRow>
-          {year}
+    <>
+      <NationalClassesDownload nationalClasses={nationalClasses} year={year} />
```

Comment:
ButtonTableExport only works with table elements, and we are using the new DataGrid for NationalClasses, which is composed of divs

Reply:
Sure I will put it there once I finish. It will take some time to get the logic right, we have some special cases with the new DataGrid, the multiselect cells would be an example.